config/runtimes/kserve-tensorflow-serving-gpu.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-tensorflow-serving-gpu
```

Comment:
the gpu runtime can be consolidated, the image tag is different.

for cpu it is `tensorflow-serving:1.14` while gpu tag is `tensorflow-serving:1.14-gpu`